web/webkit/src/main/scala/net/liftweb/builtin/snippet/Msg.scala

```diff
@@ -116,13 +116,13 @@ object Msg extends DispatchSnippet {
   }
 
   /**
-   * This method renders a <script/> element that renders effects for
+   * This method appends a script element to lift's page script that renders effects for
```

Comment:
Maybe “appends JavaScript to the current page's page script that…” rather than “appends a script element”? Definitely it's not appending an element.